crates/tools/trust_db_util.rs

```diff
@@ -257,9 +270,11 @@ fn find(opts: SearchDbOpts, _: &cfg::All, env: &Environment) -> Result<(), Error
 fn dump(opts: DumpDbOpts, cfg: &cfg::All) -> Result<(), Error> {
     let db = read::load_trust_db(&cfg.system.trust_db_path)?;
     match opts.outfile {
-        None => db
-            .iter()
-            .for_each(|(k, v)| println!("{} {} {}", k, v.trusted.size, v.trusted.hash)),
+        None => {
+            for (k, v) in db.iter() {
+                println!("{} {} {}", k, v.trusted.size, v.trusted.hash)
+            }
+        }
```

Comment:
Why was this change needed?

Reply:
It wasn't needed, just matching the style of the other case